src/routes/docs/advanced/platform/shortcuts/+page.markdoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: article
+title: Keyboard shortcuts
+description: Learn to navigate the Appwrite Console efficiently and effectively with your keyboard
+---
+
+The Appwrite Console was designed with a keyboard first approach. The Appwrite Console supports keyboard shortcuts that make it easier to navigate and perform actions quicker.
+
+# Shortcuts {% #shortcuts %}
+
+The Appwrite Console supports keyboard shortcuts that make it easier to navigate and perform common actions quicker. The shortcuts use the following pattern: use the first letter from the call to action followed by the resource, product, service, or page you're targeting.  For example, the shortcut keys `G` + `S` navigates to the project's Storage screen. Similarly, the shortcut `G` + `F` navigates to the project's Functions screen. However, when there's a conflict with a shortcut key, the following letter is used. For example, since S is already used for Storage, the shortcut for Settings is E. This pattern is used to make usage of shortcuts consistent and predictable.
+
+## Global shortcuts {% #global-shortcuts %}
+
+Developers can also utilize global shortcuts anywhere in the Appwrite Console, allowing instant access to these features from any page on the Console.
+
+|Action|Shortcut|
+|----|----|
+|Set theme to light|`T` then `L`|
+|Set theme to dark|`T` then `D`|
+|Set theme to dark|`T` then `A`|
+|Open Appwrite Assistant|`A` then `I`|
+
+## Command center
+
+All the Appwrite shortcuts can be found in the [command center](/docs/tooling/command-center).
+Use the command center to search for commands or the current page's content.
+
+|PC (Windows / Linux)|macOS|Action|
+|----|----|----|
+|`Ctrl` + `K`|`⌘` + `K`|Access Command Center|
+
+## Project shortcuts {% #project-shortcuts %}
+
+Within the context of a project, developers can utilize project shortcuts. These shortcuts are automatically enabled once a project is selected on the Console. 
+These shortcuts allow quick access to the project's database, storage, messaging, auth, settings, overview pages, and the shortcuts make the Console more accessible.
+
+|Action|Shortcut|
+|----|----|
+|Go to auth|`G` then `A`|
+|Go to storage|`G` then `S`|
+|Go to databases|`G` then `D`|
+|Go to functions|`G` then `F`|
+|Go to projects|`G` then `P`|
+|Go to overview|`G` then `O`|
+|Go to messaging|`G` then `M`|
+|Go to settings|`G` then `E`|
```

Comment:
What was the reason the items in this list were ordered this way? To me, it would make sense to order them like they are in the Console (which is also how the products ordered in the docs).